src/components/lyne-panel/lyne-panel.scss

```diff
@@ -0,0 +1,93 @@
+:host {
+  display: block;
+
+  --font-family: 'SBBWeb Light';
+  --letter-spacing: 0.01em;
+  --panel-width: 223;
+  --panel-height: 167;
+  --font-size: 24;
+  --line-height: 1.3;
+  --button-margin: var(--spacing-fixed-5x);
+  --panel-padding-inline: var(--spacing-responsive-s-medium);
```

Comment:
Bit hard to see in the current component view in Figma but I think that we need to use --spacing-responsive-m-small here.

Reply:
done